### PR TITLE
ARROW-2397: [Documentation] Update format documentation to describe tensor alignment.

### PR DESCRIPTION
**Pull request by robertnishihara**

### CODE REVIEW

File: format/IPC.md [hunk 1]
```diff
@@ -240,8 +240,8 @@ tools. Arrow implementations in general are not required to implement this data
 format, though we provide a reference implementation in C++.
 
 When writing a standalone encapsulated tensor message, we use the format as
-indicated above, but additionally align the starting offset (if writing to a
-shared memory region) to be a multiple of 8:
+indicated above, but additionally align the starting offset of the metadata as well as the starting offset of the tensor body (if writing to a shared
```
Comment (xhochy) [inline format/IPC.md]: Can you wrap the text at the same line width as above?

File: format/IPC.md [hunk 2]
```diff
@@ -240,8 +240,9 @@ tools. Arrow implementations in general are not required to implement this data
 format, though we provide a reference implementation in C++.
 
 When writing a standalone encapsulated tensor message, we use the format as
-indicated above, but additionally align the starting offset (if writing to a
-shared memory region) to be a multiple of 8:
+indicated above, but additionally align the starting offset of the metadata as
+well as the starting offset of the tensor body (if writing to a shared memory
+region) to be multiples of 64 bytes:
```
[the remark this answers is not in the document]
Comment (xhochy) [reply format/IPC.md]: From what we want we want as alignment in the Arrow standard, this is correct. I don't know if all our implementations that emit tensors conform to this.

[the remark this answers is not in the document]
Comment (robertnishihara) [reply format/IPC.md]: I see, I was just looking at 

https://github.com/apache/arrow/blob/045470c3879fd05f528dbe68f2c2299b2ebba105/cpp/src/arrow/ipc/writer.cc#L520-L528

and

https://github.com/apache/arrow/blob/045470c3879fd05f528dbe68f2c2299b2ebba105/cpp/src/arrow/ipc/util.h#L31-L36

**Comment by robertnishihara** [comment]
@xhochy should we merge this? Or should I change the number back to 8?

**Comment by xhochy** [comment]
@robertnishihara I think we can go ahead and merge this.

**Pull request closed without merging**